packages/ui/src/components/NumberInput/index.tsx

```diff
@@ -270,7 +257,7 @@ export const NumberInput = ({
   const isPlusDisabled = (maxValue && plusRoundedValue > maxValue) || disabled
 
   return (
-    <StyledContainer disabled={disabled} size={size} className={className}>
+    <StyledContainer data-disabled={disabled} size={size} className={className}>
```

Comment:
Optional : We may use `aria-disabled` for that kind of case mb and still use the attibute as css identifier

Reply:
I'm not expert, far from it, but it seems that it's better to keep `data-` as `aria-disabled` will disabled all the children, and apparently it's better to keep it for leaves elements (div with button role, etc.)
(I've used https://developer.mozilla.org/en-US/docs/Web/Accessibility/ARIA/Attributes/aria-disabled for ref)

Reply:
@fabienhebert @ModuloM In fact, we shouldn't use `aria-disabled` it should be explicit from native tags such as `button`, `input`, etc by using prop `disabled`. Here we just want to say to the styled div that the container is disabled, so I passed a `data-disabled`